<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"><html><head>  <title>HUPO Proteomics Standards Initiative Protein InteractionSpecification Documentation</title></head><body><a href="http://psidev.sourceforge.net"><img src="http://psidev.sourceforge.net/images/psi.gif" width="113" height="75" border="0" align="left"></a> <a href="http://www.hupo.org/"><img src="http://psidev.sourceforge.net/images/hupo.gif" width="113" height="75" border="0" align="right"></a> <br clear="all"><center><h2><a name="mozTocId916500" class="mozTocH2"></a><a href="http://psidev.sourceforge.net">Proteomics Standards Initiative</a><br></h2><h2><a name="mozTocId992260" class="mozTocH2"></a>Molecular InteractionXML Format Documentation</h2><h2><a name="mozTocId863651" class="mozTocH2"></a>Version 1.0</h2><div style="text-align: left;"><h2>Table of Contents</h2><ol>  <li><a href="#intro">Introduction</a></li>  <li><a href="#mi-purpose">Purpose of the PSI MI XML format</a></li>  <li><a href="#document-purpose">Purpose of this document</a></li>  <li><a href="#structure">Structure of a PSI MI record</a></li>  <li><a href="#detailed-doc">Detailed Documentation</a></li>  <li><a href="#cv">Use of external controlled vocabularies<br>    </a></li>  <li><a href="#planned">List of planned features</a></li>  <li><a href="#comment">How to comment</a></li>  <li><a href="#data">Available data<br>    </a></li>  <li><a href="#tools">Tools</a></li>  <li><a href="#submission">Data submission</a><br>  </li>  <li><a href="#further-info">Further information and relevant links</a></li></ol></div></center><h2><a name="intro" class="mozTocH2"></a>Introduction</h2>The Proteomics Standards Initiative (PSI) aims to define communitystandards for data representation in proteomics to facilitate datacomparison, exchange and verification.<p> The Proteomics Standards Initiative was founded at the HUPO meetingin Washington, April 28-29, 2002 (see <a href="http://www.sciencemag.org/cgi/content/summary/296/5569/827?maxtoshow=&amp;HITS=10&amp;hits=10&amp;RESULTFORMAT=&amp;searchid=1027414274815_1591&amp;stored_search=&amp;FIRSTINDEX=0&amp;volume=296&amp;firstpage=827&amp;fdate=10/1/1995&amp;tdate=7/31/2002">Science296, 827</a>). As a first step, the PSI is developing standards for twokey areas of proteomics: <a href="http://psidev.sourceforge.net#ms">massspectrometry</a> and <a href="http://psidev.sourceforge.net#ppi">protein-proteininteraction</a> data. </p><p> This document decribes the molecular interaction data exchangeformat. PSI is following a leveled approach to building thisspecification. Level 1 will describe protein interactions at a basiclevel that covers a large amount of currently available data.Subsequentlevels will add capability to represent new molecular interactioninformation that the community wishes to exchange. </p><p> The scope of PSI MI is currently limited to protein-proteininteractions. Other molecules, such as small molecules, DNA and RNA maybe taken into account in the future.</p><p> PSI MI was designed by a group of people including representativesfrom database providers and users in both academia and industry. PSI MIis expected to be supported by the DIP, MINT, IntAct and BIND databases.<br></p><h2><a name="mi-purpose" class="mozTocH2"></a>Purpose of the PSI MI XMLformat</h2><p>The PSI MI format is a data exchange format for protein-proteininteractions. It is not a proposed database structure. Intended usagesare described by the <a href="useCases.html">use cases documentation</a>.These use case descriptions also provide hints for future tools to bedeveloped.<br></p><p> </p><h2><a name="document-purpose" class="mozTocH2"></a>Purpose of thisdocument</h2>The purpose of this document is to describe the general structure ofthe PSI MI XML specification in a more user-friendly manner than thespecification does itself. For the detailed and most up-to-datedescription please see the <a href="../MIF.html">auto-generateddocumentation</a>. This documentation also provides additionalinformation, e.g. sample data files and use case descriptions.<br><p> </p><h2><a name="structure" class="mozTocH2"></a>Structure of a PSI MIrecord</h2>The root element of a PSI MI XML file is the entrySet. An entrySetcontains one or more entries. Each entry is a self-contained unit. Thisallows to easily concatenate the contents of multiple files into asingle file by simply adding all the entries into the entrySet.<br><br><img src="MIF_topLevel.png" title="" alt="" style="width: 354px; height: 380px;"><br><span style="font-weight: bold;">Figure 1: The entry top level element</span><br><br>Each entry describes one or more protein interactions. The PSI MIformat can be used in two forms, a compact and an expanded form. In thecompact form, all interactors (proteins), experiments, and availabilitystatements are described once in the respective list elements, and thenonly referred to by references from the individual interactions in theinteractionList. The compact form allows a dense, non-repetitiverepresentation of the data, in particular for large data sets. <br>In the expanded form, all proteins, experiments, and availabilitystatements are described directly in the interaction element. As aresult, each interaction is a self-contained element providing allnecessary information. The expanded form results in larger files, butismore suitable for conversion to displayed data, e.g. HTML pages. ThePSIMI consortium provides tools to convert the compact into the expandedform and back.<br><br>In the next sections, the top level elements shown in Figure 1 andtheir function will be described. <br><br>The source element describes the source of the entry, usually theorganisation which provides it. It also contains a release (number) anda releaseDate. <br><br>The availabilityList provides statements on the availability of thedata, usually copyright statements. In the current version, theavailability statements are free text. The PSI MI format might later beextended to provide predefined availability statements. <br><br>The experimentList contains experimentDescriptions. EachexperimentDescription describes one set of experimental parameters,usually associated with a single publication. In large-scaleexperiments, normally only one parameter is varied across a series ofexperiments, usually the bait. The PSI MI format describes the constantparameters, e.g. experimental techniques, in an experimentDescription,while the variable parameters, e.g. the bait, are described in theinteraction element. <br><br>The interactorList describes a set of interactors participating in aninteraction. In the current version of the PSI MI standard, interactorsare proteins. It is planned to extend this to other types, for examplesmall molecules, in future versions. The interactor element describesthe "normal" form of a protein, consisting of the "administrative" datalike name and crossreferences, and organism and amino acid sequence.Attributes which are relevant for a specific interaction, in particularsequence features, are described in the participant element within aninteraction. <br><br><img src="MIF_interaction.png" title="" alt="" style="width: 653px; height: 719px;"><br><span style="font-weight: bold;">Figure 2: Interaction element<br><br></span> The interactionList contains one or more interaction elements.Each interaction contains a description of the data availability(copyright), and a desciption of the experimental conditions underwhichit has been determined. Both of these can either be integrated into theinteraction element (expanded form) or refer to the respective lists inthe entry (compact form) as described above. <br><br><img src="MIF_participant.png" title="" alt="participant element" style="width: 419px; height: 429px;"><br><span style="font-weight: bold;">Figure 3: Participant element</span><br><br>Each interaction contains two or more participants, the moleculesparticipating in the interaction. Each participant element contains adescription of the molecule in its "normal" form, either by referencetoan element of the interactorList, or directly in an interactor element.<br>Additional elements of the participant element describe the specificform of the molecule in which it participated in the interaction. ThefeatureList describes sequence features of the protein, for examplebinding domains relevant for the interaction. The role describes theparticular role of the protein in the experiment, usually whether theprotein was a bait or prey.<br><br>The attributeLists are placeholders for semi-structured additional datathe data provider might want to transmit. They contain simple tag-valuepairs.<br><br><h2><a name="detailed-doc" class="mozTocH2"></a><a href="../MIF.html">DetailedDocumentation</a></h2>see <a href="http://psidev.sourceforge.net/mi/xml/doc/MIF.html">http://psidev.sourceforge.net/mi/xml/doc/MIF.html</a><br><h2><a name="cv" class="mozTocH2"></a>Use of external controlledvocabularies<br></h2>Where possible, external controlled vocabularies are referenced fromPSI MI. External controlled vocabularies are used in two forms: <br><ul>  <li><span style="font-weight: bold;">Open</span> controlledvocabularies: We think that no existing controlled vocabulary providesall necessary terms for the given attribute in the PSI MI format. Inthis case, it is up to the data provider to choose a controlledvocabulary, or to provide a free text string if no appropriatecontrolled vocabulary exists.</li>  <li><span style="font-weight: bold;">Closed </span>controlledvocabularies: We think that there is a controlled vocabulary whichappropriately covers all necessary terms for the given attribute. Inthis case, only terms from the defined vocabulary should be used.</li></ul>The following closed controlled vocabularies are referenced by PSI MI:<br><ul>  <li><span lang="EN-US" style="">interaction type</span></li>  <li><span lang="EN-US" style="">sequencefeature type</span></li>  <li><span lang="EN-US" style="">feature detection</span></li>  <li><span lang="EN-US" style="">participant detection</span></li>  <li><span lang="EN-US" style="">interaction detection<o:p></o:p></span>  </li></ul>These CVs are grouped together in one pair of *.dag (hierarchy) and*.def (definitions) files in <a href="http://www.geneontology.org">GeneOntology</a> flat file format. (<a href="../../../controlledVocab/">allfiles</a>, GO format: <a href="../../../controlledVocab/psi-mi.dag">psi-mi.dag</a>,<a href="../../../controlledVocab/psi-mi.def">psi-mi.def</a>, <a href="../../../controlledVocab/psi-mi.dag.html">HTML version</a> of GOformat). <br>The correctness of references to external controlled vocabularies iscurrently not enforced by the PSI MI schema. It is the responsibilityofthe data provider to ensure that only existing terms at an up-to-datedata source are referenced. The World Wide Web Consortium (W3C) hasrecently issued a new <a href="http://www.w3.org/TR/2003/REC-xptr-framework-20030325/">recommendationfor referencing between XML documents</a>. Once this recommendation isimplemented by standard software, we will include it in the PSI MIschema.<br><h2><a name="planned" class="mozTocH2"></a>List of planned features</h2>Because we are following a leveled approach, we are interested inknowing what the community wishes to be included in the next level.<p> The following items have been tagged for inclusion in the nextlevel: </p><ul>  <li>Intramolecular interactions</li>  <li>Inclusion of other molecule types, e.g. DNA, RNA, small molecules<br>  </li></ul>The latest list of features to discuss/include in the future can befound here:<br><a href="http://sourceforge.net/tracker/?atid=511101&amp;group_id=65472&amp;func=browse">http://sourceforge.net/tracker/?atid=511101&amp;group_id=65472&amp;func=browse</a><br><h2><a name="comment" class="mozTocH2"></a>How to comment</h2>If you would like to comment on this document, the PSI MI XMLspecification, please send a mail to:<br><a href="mailto:psidev-mi-dev@lists.sourceforge.net">psidev-mi-dev@lists.sourceforge.net</a>&nbsp;<h2><a name="data" class="mozTocH2"></a>Available data</h2><ul>  <li>The <a href="../../sampleData/index.html">sample files</a>contain authentic data, though we don't take any responsibility for thecorrectness of the data. </li>  <li><a href="http://www.hprd.org">HPRD</a> provides PSI MI formatteddownloads of interaction data by adding /psimi to a standard moleculeview URL. Sample URL:<br>    <a class="moz-txt-link-freetext" href="http://www.hprd.org/protein/00150/psimi">http://www.hprd.org/protein/00150/psimi</a></li>  <li><a href="http://www.hybrigenics.com">Hybrigenics </a>H.pyloridataset from <br>    <cite>Rain JC, Selig L, De Reuse H, Battaglia V, Reverdy C, SimonS, Lenzen G, Petel F, Wojcik J, Schachter V, Chemama Y, Labigne A,Legrain P.: The protein-protein interaction map of Helicobacter pylori.Nature. 2001 Jan 11;409(6817):211-5. PMID: <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11196647&amp;dopt=Abstract">11196647</a>.</cite><br>Available after registration from <a href="http://pim.hybrigenics.com">http://pim.hybrigenics.com</a>.</li>  <li><a href="http://www.ebi.ac.uk/intact">IntAct</a> offers PSI MI data files at <a href="ftp://ftp.ebi.ac.uk:/pub/databases/intact/current/xml/">ftp://ftp.ebi.ac.uk:/pub/databases/intact/current/xml</a> and dynamically generated PSI MI data: A graphical interaction network viewer allows to download thedisplayed network in PSI MI format, for example for further analysis inone of the tools listed below. Sample URL:<br>    <a class="moz-txt-link-freetext" href="http://www.ebi.ac.uk/intact/graph2mif/getXML?ac=EBI-367&amp;depth=1&amp;strict=false">http://www.ebi.ac.uk/intact/graph2mif/getXML?ac=EBI-367&amp;depth=1&amp;strict=false</a></li>  <li><a href="http://cbm.bio.uniroma2.it/mint/">MINT</a> offers thecomplete dataset in PSI MI format.<br>  </li>  <li><a href="http://dip.doe-mbi.ucla.edu/dip/Download.cgi">DIP</a> offers thecomplete dataset in PSI MI format.<br>  </li></ul><h2><a name="tools" class="mozTocH2"></a>Tools</h2>PSI MI XML format is supported by a growing list of tools. Currentlyavailable are:<br><ul>  <li><a href="../../src/MIF_compact.xsl">MIF_compact.xsl</a>:Transforms a file from the expanded into the compact form.</li>  <li><a href="../../src/MIF_expand.xsl">MIF_expand.xsl</a>: Transformsa file from the compact into the expanded form.<br>  </li>  <li><a href="../../src/MIF_view.xsl">MIF_view.xsl</a>: Creates asimple HTML view of the PSI-MI file. This does not show all details ofthe PSI-MI record.</li>  <li><a href="http://www.cytoscape.org">Cytoscape</a>: Tool foranalysis of interaction networks with <a href="http://www.cytoscape.org/plugins2.php">plugin for PSI reading</a>.</li>  <li><a href="http://pim.hybrigenics.com/pimwalker/">PIMWalker</a>:Interaction network viewer with PSI reading.</li>  <li><a href="../../../tools/psimakers0908src.zip">Transformation tools</a>to convert PSI MI XML format into tab-delimited ASCII format and viceversa.</li></ul><h2><a name="submission" class="mozTocH2"></a>Data submission</h2>The following databases currently accept submissions of PSI MIformatted interaction data:<br><ul>  <li><a href="http://www.bind.ca">BIND:</a> Contact and submission byemail: <big><font size="2"><big><a href="mailto:info@bind.ca"><font color="#0000ff">info@bind.ca</font></a></big></font></big></li>  <li><a href="http://dip.doe-mbi.ucla.edu/">DIP</a><big><font size="2"><big><font color="#0000ff">:</font></big></font></big> Contact and submission byemail: <a href="mailto:dip@mbi.ucla.edu">dip@mbi.ucla.edu</a></li>  <li><a href="http://www.hprd.org">HPRD</a>: Web-based submissiontool: <a href="http://psimi.ibioinformatics.org/">http://psimi.ibioinformatics.org/</a><br>  </li>  <li><a href="http://www.ebi.ac.uk/intact">IntAct:</a> Contact andsubmission by email: <a href="mailto:datasubs@ebi.ac.uk">datasubs@ebi.ac.uk</a><br>  </li>  <li><a href="http://cbm.bio.uniroma2.it/mint/">MINT:</a> Contact andsubmission by email: <a href="mailto:cesareni@uniroma2.it">cesareni@uniroma2.it</a></li></ul><h2><a name="further-info" class="mozTocH2"></a>Further information andrelevant links</h2><p> Databases involved: </p><ul>  <li><a href="http://www.bind.ca">BIND </a></li>  <li><a href="http://dip.doe-mbi.ucla.edu/">DIP</a></li>  <li><a href="http://www.hprd.org">HPRD</a></li>  <li><a href="http://www.ebi.ac.uk/intact">IntAct </a></li>  <li><a href="http://cbm.bio.uniroma2.it/mint/">MINT </a></li></ul>Companies involved:<ul>  <li><a href="http://www.hybrigenics.com/">Hybrigenics</a></li>  <li><a href="http://www.cellzome.com/">Cellzome</a><br>  </li></ul>Related Efforts:<ul>  <li><a href="http://biopax.org">BioPAX</a></li></ul><hr style="width: 100%; height: 2px;"><br></body></html>