<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>
  <title>HUPO Proteomics Standards Initiative Protein Interaction
Specification Documentation</title>
</head>
<body>
<a href="http://psidev.sourceforge.net"><img
 src="http://psidev.sourceforge.net/images/psi.gif" width="113"
 height="75" border="0" align="left"></a> <a
 href="http://www.hupo.org/"><img
 src="http://psidev.sourceforge.net/images/hupo.gif" width="113"
 height="75" border="0" align="right"></a> <br clear="all">
<center>
<h2><a name="mozTocId916500" class="mozTocH2"></a><a
 href="http://psidev.sourceforge.net">Proteomics Standards Initiative</a>
<br>
</h2>
<h2><a name="mozTocId992260" class="mozTocH2"></a>Molecular Interaction
XML Format Documentation</h2>
<h2><a name="mozTocId863651" class="mozTocH2"></a>Version 1.0</h2>
<div style="text-align: left;">
<h2>Table of Contents</h2>
<ol>
  <li><a href="#intro">Introduction</a></li>
  <li><a href="#mi-purpose">Purpose of the PSI MI XML format</a></li>
  <li><a href="#document-purpose">Purpose of this document</a></li>
  <li><a href="#structure">Structure of a PSI MI record</a></li>
  <li><a href="#detailed-doc">Detailed Documentation</a></li>
  <li><a href="#cv">Use of external controlled vocabularies<br>
    </a></li>
  <li><a href="#planned">List of planned features</a></li>
  <li><a href="#comment">How to comment</a></li>
  <li><a href="#data">Available data<br>
    </a></li>
  <li><a href="#tools">Tools</a></li>
  <li><a href="#submission">Data submission</a><br>
  </li>
  <li><a href="#further-info">Further information and relevant links</a></li>
</ol>
</div>
</center>
<h2><a name="intro" class="mozTocH2"></a>Introduction</h2>
The Proteomics Standards Initiative (PSI) aims to define community
standards for data representation in proteomics to facilitate data
comparison, exchange and verification.
<p> The Proteomics Standards Initiative was founded at the HUPO meeting
in Washington, April 28-29, 2002 (see <a
 href="http://www.sciencemag.org/cgi/content/summary/296/5569/827?maxtoshow=&amp;HITS=10&amp;hits=10&amp;RESULTFORMAT=&amp;searchid=1027414274815_1591&amp;stored_search=&amp;FIRSTINDEX=0&amp;volume=296&amp;firstpage=827&amp;fdate=10/1/1995&amp;tdate=7/31/2002">Science
296, 827</a>). As a first step, the PSI is developing standards for two
key areas of proteomics: <a href="http://psidev.sourceforge.net#ms">mass
spectrometry</a> and <a href="http://psidev.sourceforge.net#ppi">protein-protein
interaction</a> data. </p>
<p> This document decribes the molecular interaction data exchange
format. PSI is following a leveled approach to building this
specification. Level 1 will describe protein interactions at a basic
level that covers a large amount of currently available data.
Subsequent
levels will add capability to represent new molecular interaction
information that the community wishes to exchange. </p>
<p> The scope of PSI MI is currently limited to protein-protein
interactions. Other molecules, such as small molecules, DNA and RNA may
be taken into account in the future.</p>
<p> PSI MI was designed by a group of people including representatives
from database providers and users in both academia and industry. PSI MI
is expected to be supported by the DIP, MINT, IntAct and BIND databases.<br>
</p>
<h2><a name="mi-purpose" class="mozTocH2"></a>Purpose of the PSI MI XML
format</h2>
<p>The PSI MI format is a data exchange format for protein-protein
interactions. It is not a proposed database structure. Intended usages
are described by the <a href="useCases.html">use cases documentation</a>.
These use case descriptions also provide hints for future tools to be
developed.<br>
</p>
<p> </p>
<h2><a name="document-purpose" class="mozTocH2"></a>Purpose of this
document</h2>
The purpose of this document is to describe the general structure of
the PSI MI XML specification in a more user-friendly manner than the
specification does itself. For the detailed and most up-to-date
description please see the <a href="../MIF.html">auto-generated
documentation</a>. This documentation also provides additional
information, e.g. sample data files and use case descriptions.<br>
<p> </p>
<h2><a name="structure" class="mozTocH2"></a>Structure of a PSI MI
record</h2>
The root element of a PSI MI XML file is the entrySet. An entrySet
contains one or more entries. Each entry is a self-contained unit. This
allows to easily concatenate the contents of multiple files into a
single file by simply adding all the entries into the entrySet.<br>
<br>
<img src="MIF_topLevel.png" title="" alt=""
 style="width: 354px; height: 380px;"><br>
<span style="font-weight: bold;">Figure 1: The entry top level element</span><br>
<br>
Each entry describes one or more protein interactions. The PSI MI
format can be used in two forms, a compact and an expanded form. In the
compact form, all interactors (proteins), experiments, and availability
statements are described once in the respective list elements, and then
only referred to by references from the individual interactions in the
interactionList. The compact form allows a dense, non-repetitive
representation of the data, in particular for large data sets. <br>
In the expanded form, all proteins, experiments, and availability
statements are described directly in the interaction element. As a
result, each interaction is a self-contained element providing all
necessary information. The expanded form results in larger files, but
is
more suitable for conversion to displayed data, e.g. HTML pages. The
PSI
MI consortium provides tools to convert the compact into the expanded
form and back.<br>
<br>
In the next sections, the top level elements shown in Figure 1 and
their function will be described. <br>
<br>
The source element describes the source of the entry, usually the
organisation which provides it. It also contains a release (number) and
a releaseDate. <br>
<br>
The availabilityList provides statements on the availability of the
data, usually copyright statements. In the current version, the
availability statements are free text. The PSI MI format might later be
extended to provide predefined availability statements. <br>
<br>
The experimentList contains experimentDescriptions. Each
experimentDescription describes one set of experimental parameters,
usually associated with a single publication. In large-scale
experiments, normally only one parameter is varied across a series of
experiments, usually the bait. The PSI MI format describes the constant
parameters, e.g. experimental techniques, in an experimentDescription,
while the variable parameters, e.g. the bait, are described in the
interaction element. <br>
<br>
The interactorList describes a set of interactors participating in an
interaction. In the current version of the PSI MI standard, interactors
are proteins. It is planned to extend this to other types, for example
small molecules, in future versions. The interactor element describes
the "normal" form of a protein, consisting of the "administrative" data
like name and crossreferences, and organism and amino acid sequence.
Attributes which are relevant for a specific interaction, in particular
sequence features, are described in the participant element within an
interaction. <br>
<br>
<img src="MIF_interaction.png" title="" alt=""
 style="width: 653px; height: 719px;"><br>
<span style="font-weight: bold;">Figure 2: Interaction element<br>
<br>
</span> The interactionList contains one or more interaction elements.
Each interaction contains a description of the data availability
(copyright), and a desciption of the experimental conditions under
which
it has been determined. Both of these can either be integrated into the
interaction element (expanded form) or refer to the respective lists in
the entry (compact form) as described above. <br>
<br>
<img src="MIF_participant.png" title="" alt="participant element"
 style="width: 419px; height: 429px;"><br>
<span style="font-weight: bold;">Figure 3: Participant element</span><br>
<br>
Each interaction contains two or more participants, the molecules
participating in the interaction. Each participant element contains a
description of the molecule in its "normal" form, either by reference
to
an element of the interactorList, or directly in an interactor element.
<br>
Additional elements of the participant element describe the specific
form of the molecule in which it participated in the interaction. The
featureList describes sequence features of the protein, for example
binding domains relevant for the interaction. The role describes the
particular role of the protein in the experiment, usually whether the
protein was a bait or prey.<br>
<br>
The attributeLists are placeholders for semi-structured additional data
the data provider might want to transmit. They contain simple tag-value
pairs.<br>
<br>
<h2><a name="detailed-doc" class="mozTocH2"></a><a href="../MIF.html">Detailed
Documentation</a></h2>
see <a href="http://psidev.sourceforge.net/mi/xml/doc/MIF.html">http://psidev.sourceforge.net/mi/xml/doc/MIF.html</a><br>
<h2><a name="cv" class="mozTocH2"></a>Use of external controlled
vocabularies<br>
</h2>
Where possible, external controlled vocabularies are referenced from
PSI MI. External controlled vocabularies are used in two forms: <br>
<ul>
  <li><span style="font-weight: bold;">Open</span> controlled
vocabularies: We think that no existing controlled vocabulary provides
all necessary terms for the given attribute in the PSI MI format. In
this case, it is up to the data provider to choose a controlled
vocabulary, or to provide a free text string if no appropriate
controlled vocabulary exists.</li>
  <li><span style="font-weight: bold;">Closed </span>controlled
vocabularies: We think that there is a controlled vocabulary which
appropriately covers all necessary terms for the given attribute. In
this case, only terms from the defined vocabulary should be used.</li>
</ul>
The following closed controlled vocabularies are referenced by PSI MI:<br>
<ul>
  <li><span lang="EN-US" style="">interaction type</span></li>
  <li><span lang="EN-US" style="">sequence
feature type</span></li>
  <li><span lang="EN-US" style="">feature detection</span></li>
  <li><span lang="EN-US" style="">participant detection</span></li>
  <li><span lang="EN-US" style="">interaction detection<o:p></o:p></span>
  </li>
</ul>
These CVs are grouped together in one pair of *.dag (hierarchy) and
*.def (definitions) files in <a href="http://www.geneontology.org">Gene
Ontology</a> flat file format. (<a href="../../../controlledVocab/">all
files</a>, GO format: <a href="../../../controlledVocab/psi-mi.dag">psi-mi.dag</a>,
<a href="../../../controlledVocab/psi-mi.def">psi-mi.def</a>, <a
 href="../../../controlledVocab/psi-mi.dag.html">HTML version</a> of GO
format). <br>
The correctness of references to external controlled vocabularies is
currently not enforced by the PSI MI schema. It is the responsibility
of
the data provider to ensure that only existing terms at an up-to-date
data source are referenced. The World Wide Web Consortium (W3C) has
recently issued a new <a
 href="http://www.w3.org/TR/2003/REC-xptr-framework-20030325/">recommendation
for referencing between XML documents</a>. Once this recommendation is
implemented by standard software, we will include it in the PSI MI
schema.<br>
<h2><a name="planned" class="mozTocH2"></a>List of planned features</h2>
Because we are following a leveled approach, we are interested in
knowing what the community wishes to be included in the next level.
<p> The following items have been tagged for inclusion in the next
level: </p>
<ul>
  <li>Intramolecular interactions</li>
  <li>Inclusion of other molecule types, e.g. DNA, RNA, small molecules<br>
  </li>
</ul>
The latest list of features to discuss/include in the future can be
found here:<br>
<a
 href="http://sourceforge.net/tracker/?atid=511101&amp;group_id=65472&amp;func=browse">http://sourceforge.net/tracker/?atid=511101&amp;group_id=65472&amp;func=browse</a><br>
<h2><a name="comment" class="mozTocH2"></a>How to comment</h2>
If you would like to comment on this document, the PSI MI XML
specification, please send a mail to:<br>
<a href="mailto:psidev-mi-dev@lists.sourceforge.net">psidev-mi-dev@lists.sourceforge.net</a>&nbsp;
<h2><a name="data" class="mozTocH2"></a>Available data</h2>
<ul>
  <li>The <a href="../../sampleData/index.html">sample files</a>
contain authentic data, though we don't take any responsibility for the
correctness of the data. </li>
  <li><a href="http://www.hprd.org">HPRD</a> provides PSI MI formatted
downloads of interaction data by adding /psimi to a standard molecule
view URL. Sample URL:<br>
    <a class="moz-txt-link-freetext"
 href="http://www.hprd.org/protein/00150/psimi">http://www.hprd.org/protein/00150/psimi</a></li>
  <li><a href="http://www.hybrigenics.com">Hybrigenics </a>H.pylori
dataset from <br>
    <cite>Rain JC, Selig L, De Reuse H, Battaglia V, Reverdy C, Simon
S, Lenzen G, Petel F, Wojcik J, Schachter V, Chemama Y, Labigne A,
Legrain P.: The protein-protein interaction map of Helicobacter pylori.
Nature. 2001 Jan 11;409(6817):211-5. PMID: <a
 href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11196647&amp;dopt=Abstract">11196647</a>.</cite><br>
Available after registration from <a href="http://pim.hybrigenics.com">http://pim.hybrigenics.com</a>.</li>
  <li><a href="http://www.ebi.ac.uk/intact">IntAct</a> offers PSI MI data files at <a href="ftp://ftp.ebi.ac.uk:/pub/databases/intact/current/xml/">ftp://ftp.ebi.ac.uk:/pub/databases/intact/current/xml</a> and dynamically generated PSI MI data: A graphical interaction network viewer allows to download the
displayed network in PSI MI format, for example for further analysis in
one of the tools listed below. Sample URL:<br>
    <a class="moz-txt-link-freetext"
 href="http://www.ebi.ac.uk/intact/graph2mif/getXML?ac=EBI-367&amp;depth=1&amp;strict=false">http://www.ebi.ac.uk/intact/graph2mif/getXML?ac=EBI-367&amp;depth=1&amp;strict=false</a></li>
  <li><a href="http://cbm.bio.uniroma2.it/mint/">MINT</a> offers the
complete dataset in PSI MI format.<br>
  </li>
  <li><a href="http://dip.doe-mbi.ucla.edu/dip/Download.cgi">DIP</a> offers the
complete dataset in PSI MI format.<br>
  </li>
</ul>
<h2><a name="tools" class="mozTocH2"></a>Tools</h2>
PSI MI XML format is supported by a growing list of tools. Currently
available are:<br>
<ul>
  <li><a href="../../src/MIF_compact.xsl">MIF_compact.xsl</a>:
Transforms a file from the expanded into the compact form.</li>
  <li><a href="../../src/MIF_expand.xsl">MIF_expand.xsl</a>: Transforms
a file from the compact into the expanded form.<br>
  </li>
  <li><a href="../../src/MIF_view.xsl">MIF_view.xsl</a>: Creates a
simple HTML view of the PSI-MI file. This does not show all details of
the PSI-MI record.</li>
  <li><a href="http://www.cytoscape.org">Cytoscape</a>: Tool for
analysis of interaction networks with <a
 href="http://www.cytoscape.org/plugins.html">plugin for PSI reading</a>.</li>
  <li><a href="http://pim.hybrigenics.com/pimwalker/">PIMWalker</a>:
Interaction network viewer with PSI reading.</li>
  <li><a href="../../../tools/psimakers0908src.zip">Transformation tools</a>
to convert PSI MI XML format into tab-delimited ASCII format and vice
versa.</li>
</ul>
<h2><a name="submission" class="mozTocH2"></a>Data submission</h2>
The following databases currently accept submissions of PSI MI
formatted interaction data:<br>
<ul>
  <li><a href="http://www.bind.ca">BIND:</a> Contact and submission by
email: <big><font size="2"><big><a href="mailto:info@bind.ca"><font
 color="#0000ff">info@bind.ca</font></a></big></font></big></li>
  <li><a href="http://dip.doe-mbi.ucla.edu/">DIP</a><big><font size="2"><big><font
 color="#0000ff">:</font></big></font></big> Contact and submission by
email: <a href="mailto:dip@mbi.ucla.edu">dip@mbi.ucla.edu</a></li>
  <li><a href="http://www.hprd.org">HPRD</a>: Web-based submission
tool: <a href="http://psimi.ibioinformatics.org/">http://psimi.ibioinformatics.org/</a><br>
  </li>
  <li><a href="http://www.ebi.ac.uk/intact">IntAct:</a> Contact and
submission by email: <a href="mailto:datasubs@ebi.ac.uk">datasubs@ebi.ac.uk</a><br>
  </li>
  <li><a href="http://cbm.bio.uniroma2.it/mint/">MINT:</a> Contact and
submission by email: <a href="mailto:cesareni@uniroma2.it">cesareni@uniroma2.it</a></li>
</ul>
<h2><a name="further-info" class="mozTocH2"></a>Further information and
relevant links</h2>
<p> Databases involved: </p>
<ul>
  <li><a href="http://www.bind.ca">BIND </a></li>
  <li><a href="http://dip.doe-mbi.ucla.edu/">DIP</a></li>
  <li><a href="http://www.hprd.org">HPRD</a></li>
  <li><a href="http://www.ebi.ac.uk/intact">IntAct </a></li>
  <li><a href="http://cbm.bio.uniroma2.it/mint/">MINT </a></li>
</ul>
Companies involved:
<ul>
  <li><a href="http://www.hybrigenics.com/">Hybrigenics</a></li>
  <li><a href="http://www.cellzome.com/">Cellzome</a><br>
  </li>
</ul>
Related Efforts:
<ul>
  <li><a href="http://biopax.org">BioPAX</a></li>
</ul>
<hr style="width: 100%; height: 2px;"><br>
</body>
</html>
